lib/line/bot.rb

```diff
@@ -23,6 +23,7 @@
 
 # V2
 require 'line/bot/v2/webhook_parser'
+require 'line/bot/v2/version'
```

Comment:
Personally, I think it may be good to write `require ...` where `Line::Bot::V2::VERSION` is used. In other words, I think it may be good to write the `require ...` in `lib/line/bot/v2/http_client.rb`. What do you think?